shared/templates/forgot_password.html

```diff
@@ -130,8 +130,6 @@
                             </tr>
                           </tbody>
                         </table>
-                        <p class="more-info" style="color: #f2f2f2; font-family: sans-serif; font-weight: 400; line-height: 22px; letter-spacing: 0px; text-align: center; margin: 0; margin-bottom: 15px; padding-bottom: 16px; font-size: 14px;">Se tiver problemas, copie e cole o link abaixo em outra janela do seu navegador:</p>
-                        <p class="url-link" style="font-family: sans-serif; font-weight: 400; line-height: 22px; letter-spacing: 0px; margin: 0; margin-bottom: 15px; padding-bottom: 16px; text-align: center; font-size: 14px; color: #763DD3; text-decoration: none!important;;"><a class="url-link" href="{{.Url}}">{{.Url}}</a></p>
```

Comment:
more-info and url-link won't be used anymore, what about remove them from the tag style?

Reply:
Good point

Reply:
`more-info` don't exist on this file, only `url-link`